examples/kubeflow/main_ocp.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
`2015 The TensorFlow Authors` ->  `2020 The Kubeflow Authors `

Please also add the License information to new Python script in the PR. Thanks